laalaa/apps/categories/views.py

```diff
@@ -0,0 +1,10 @@
+from rest_framework import viewsets
+
+from .serializers import CategorySerializer
+from advisers.models import Category
+
+
+class CategoryViewSet(viewsets.ReadOnlyModelViewSet):
```

Comment:
Sorry. 🙇I started a few conversations elsewhere about the categories of law. That made me think about this pull request and raised some questions for me:

- What do you think about calling them `/categories_of_law` instead of `/categories`?
- What do you think about adding a test that asserts that an HTTP GET request to that URL will result in a JSON object that contains one of the elements of the constants?

Reply:
I think these are good suggestions and will look to them to the PR

Reply:
updated `/categories` path to `/categories_of_law` and added test